datafusion/physical-plan/src/aggregates/order/partial.rs

```diff
@@ -22,6 +22,7 @@ use datafusion_common::Result;
 use datafusion_execution::memory_pool::proxy::VecAllocExt;
 use datafusion_expr::EmitTo;
 use datafusion_physical_expr::PhysicalSortExpr;
+use std::mem::size_of;
```

Comment:
These could be avoided, but are OK to have.

Reply:
Not sure what you mean here

I think newer versions of rust automatically import `size_of` but whatever Rust version we have defined as the minimum supported version still requires an explicit import here

Reply:
Yes, the minimum supported version test was failing without the explicit import.